### DISCUSSION TITLE
Error: Cannot find module 'babel-plugin-import'

### DISCUSSION THREAD

**Discussion by nodeworks** [vercel/next.js#19640] [Help] [answered]
Upgrading from Next 10.0.1 -> 10.0.2 causes dev to break:

```
error - /yarn/v6/npm-next-10.0.3-2bf9a1625dcd0afc8c31be19fc5516af68d99e80-integrity/node_modules/next/dist/client/next-dev.js
Error: Cannot find module 'babel-plugin-import' from '/usr/src/app'
```

Rolling back to 10.0.1 fixes the issue. Using Yarn PnP and yarn version 1.22.4.

babel-plugin-import is included in my package.json so it's not an issue with that.

**Comment by secundant** [comment, marked as answer]
@nodeworks You can rename `.babelrc` to `.babelrc.js` and wrap all plugins/presets with `require.resolve` for fix it.
For example
```
// .babelrc
{
  "presets": ["next/babel"],
  "plugins": [
    [
      "babel-plugin-import",
      { "libraryName": "foo" }
    ]
  ]
}
// .babelrc.js
module.exports = {
  presets: [require.resolve('next/babel')],
  plugins: [
    [
      require.resolve('babel-plugin-import'),
      { libraryName: 'foo' }
    ]
  ]
}
```

**Reply by nodeworks** [reply]
Thanks @secundant that worked for me!

**Reply by ceefour** [reply]
<del>Still doesn't work for me :(</del>

I had to do this before: (seems obvious but not obvious for me)

```bash
yarn add --dev babel-plugin-import
```
